index.js

```diff
@@ -78,21 +82,11 @@ CarteroNodeHook.prototype.getAssetsForEntryPoint = function( entryPointPath, cb
 	}
 };
 
-CarteroNodeHook.prototype.getAssetUrl = function( assetSrcAbsPath ) {
+CarteroNodeHook.prototype.getAssetUrl = function( assetSrcPath, cb ) {
```

Comment:
ok no problem (for our use case we were using relative urls by modifying the metaData.json so I changed it to be less restrictive)